SHiELD/coupler_main.F90

```diff
@@ -388,12 +385,8 @@ subroutine coupler_init
 
     call print_memuse_stats('after atmos model init')
 
-    call mpp_get_global_domain(Atm%Domain, xsize=gnlon, ysize=gnlat)
-    allocate ( glon_bnd(gnlon+1,gnlat+1), glat_bnd(gnlon+1,gnlat+1) )
-    call mpp_global_field(Atm%Domain, Atm%lon_bnd, glon_bnd, position=CORNER)
-    call mpp_global_field(Atm%Domain, Atm%lat_bnd, glat_bnd, position=CORNER)
-
-    if (.NOT.Atm%bounded_domain) call data_override_init (Atm_domain_in  = Atm%domain)
```

Comment:
Do we know what the logic behind excluding bounded_domain is here? I know Lauren opened an issue a while ago relating to some unsupported regional model behavior in data override, but that could be unrelated. I would assume there wouldn't be a data_table if the bounded_domain existed because it would never get called in this scenario, but I don't know that for sure.

Reply:
@GFDL-Eric - this is a topic I'll be bringing up for FMS dev as a whole and data_override in particular.  I'll try to make it a topic for the next group meeting.